test/command_callback/python_paths/with_virtualenv/env/bin/black

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```

Comment:
Instead of copying the script into this repository, just create an empty file with the execute bit set with `chmod`. We don't need to actually run the script, we just need to see if it's there so ALE will try to run it later.